tests/suites/test_suite_ctr_drbg.function

```diff
@@ -109,15 +58,15 @@ void ctr_drbg_validate( int reseed_mode, char *nonce_string,
          *                 reseed[:reseed_len]) */
         TEST_ASSERT( mbedtls_ctr_drbg_reseed(
                          &ctx,
-                         reseed, reseed_len ) == 0 );
+                         reseed->x, reseed->len ) == 0 );
     }
 
-    /* CTR_DRBG_Generate(result_len * 8 bits, add1[:add1_len]) -> buf */
+    /* CTR_DRBG_Generate(result->len * 8 bits, add1[:add1_len]) -> buf */
```

Comment:
Please change `add1_len` to `add1->len` as well. Same thing with the other `xxx_len`.

Reply:
fixed